refactor(localpv): move builds to dynamic-localpv-provisioner repo

kmova opened the pull request:
As part of enabling multi-arch repo and deprecating the
openebs/maya repo, the local pv provisioner code is being
migrated to https://github.com/openebs/dynamic-localpv-provisioner

Disabling the travis builds on Local PV provisioner in
the openebs/maya repository

Signed-off-by: kmova <kiran.mova@mayadata.io>

akhilerm reviewed (commented):
@kmova , Some more files where localPV is referred that need to be fixed.
- https://github.com/openebs/maya/blame/master/RELEASE.md#L45
- https://github.com/openebs/maya/blob/master/tests/artifacts/openebs-ci.yaml#L579

question: will the tests pass in maya after localPV is removed. because the image wont be available since its built in a separate repo.

kmova commented:
@akhilerm - addressed the comments. The CI will work even if the images have moved to other repo. (similar to running with ndm and provisioner images)

akhilerm reviewed (approved):
/lgtm
